cairo/src/enums.rs

```diff
@@ -53,28 +53,59 @@ macro_rules! gvalue_impl {
     };
 }
 
+/// Specifies the type of antialiasing to do when rendering text or shapes.
+///
+/// `cairo 1.12` added a set of antialiasing hints, rather than specifying a specific antialias
+/// method. These hints are:
+///
+/// - [`Antialias::Fast`]: Allow the backend to degrade raster quality for speed.
+/// - [`Antialias::Good`]: Balance between speed and quality.
+/// - [`Antialias::Best`]: High-fidelity, but potentially slow, raster mode.
+///
+/// These make no guarantee on how the backend will perform its rasterisation (if it even
+/// rasterises!), nor that they have any differing effect other than to enable some form of
+/// antialiasing. In the case of glyph rendering, [`Antialias::Fast`] and [`Antialias::Good`] will
+/// be mapped to [`Antialias::Gray`], with [`Antialias::Best`] being equivalent to
+/// [`Antialias::Subpixel`].
+///
+/// The interpretation of [`Antialias::Default`] is left entirely up to the backend. Typically,
+/// this will be similar to [`Antialias::Good`].
 #[derive(Clone, PartialEq, Eq, PartialOrd, Debug, Copy)]
 #[non_exhaustive]
 #[doc(alias = "cairo_antialias_t")]
 pub enum Antialias {
+    /// Use the default antialiasing for the subsystem and target device.
```

Comment:
Do we want to add Default impls on enums that have "Default" variants like this one?

Reply:
possibly yes, as a separate commit please :)